src/test/java/org/monarchinitiative/svart/util/VcfBreakendIntegrationTest.java

```diff
@@ -28,7 +27,32 @@
             "17, 198983, bndZ, C, .C, '', event5",
     })
     public void roundTrip(String chr, int pos, String id, String ref, String alt, String mateId, String eventId) {
-        BreakendVariant variant = vcfConverter.convertBreakend(vcfConverter.parseContig(chr), id, Position.of(pos), ref, alt, ConfidenceInterval.precise(), mateId, eventId);
+        BreakendVariant variant = vcfConverter.convertBreakend(vcfConverter.parseContig(chr), id, pos, ConfidenceInterval.precise(), ref, alt, ConfidenceInterval.precise(), mateId, eventId);
+//// TODO: Want to preserve input CHROM, POS, REF, ALT
```

Comment:
@julesjacobsen Do we need to re-enable these tests? I do not think we actually need to test the `variant` attributes here, right? We test the correctness of the `vcfConverter` in `VcfBreakendResolverTest`. The point of this test seems to be the last assertion (line 56). If that's the case, we should remove the commented code. I did not want to do this without checking with you. :)

Thanks!